docs/standard-library/openapi3/openapi.md

```diff
@@ -261,6 +263,30 @@ The OpenAPI emitter ignores the "names" for variants in named unions.
 The OpenAPI emitter also defines the[`@oneOf` decorator](../openapi/reference/decorators.md#OpenAPI.oneof) which can be specified on a `union` statement to indicate
 that a union should be emitted as a `oneOf` rather than `anyOf`.
 
+## Encoding and formats
+
+When working with the `@encode` decorator the rule is as follow. Given the 3 values `encoding`, `encodeAs` and `realType` where `@encode(encoding, encodeAs) _: realType`:
+
+1. Is one of those special cases:
+
+   - `unixTimestamp` encoding will produce `type: integer, format: unixtime`
```

Comment:
Do we need to fix here?
https://github.com/microsoft/typespec/blob/main/packages/compiler/lib/decorators.ts#L664-L665

It seems validate for "string", instead of "integer".

Reply:
hhm, doesn't seem to complain either way, probably why didn't catch this earlier

Reply:
Yeah, seems the casing is the reason why no validation happen to unixTimestamp at all.

Feel free for another PR for it. It is not exactly relate to this PR on doc and openapi3.

Reply:
ah its just a typo on the casing

Reply:
yeah I'll do that in another PR as there is no change to the compiler here